dockerfiles/github-action-nightly-builder-arm/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Image for building async-profiler release packages for arm64 on github actions
+
+FROM public.ecr.aws/codebuild/amazonlinux2-aarch64-standard:3.0
+
+RUN yum update -y && yum install -y libicu tar sudo patchelf make gcc-c++ libstdc++-static java-11-amazon-corretto
```

Comment:
What is `libicu` for?

Reply:
libicu is needed for the github actions setup script needed by codebuild, will add a comment